core/sys/win32/windows.odin

```diff
@@ -19,12 +20,14 @@ Hgdiobj   :: distinct Handle;
 Hmodule   :: distinct Handle;
 Hmonitor  :: distinct Handle;
 Hrawinput :: distinct Handle;
+Hresult   :: distinct int;
```

Comment:
`HRESULT` is defined as a `long` which on windows is 32 bits. `i32` should be used, not `int`.